server/src/main/java/org/elasticsearch/search/aggregations/bucket/histogram/DateRangeHistogramAggregator.java

```diff
@@ -153,10 +151,6 @@ public void collect(int doc, long bucket) throws IOException {
         };
     }
 
-    private long offsetAwareRounding(Rounding rounding, long value, long offset) {
```

Comment:
Glad to see this getting moved into the rounding itself.  Makes much more sense.